WalletWasabi.Packager/Program.cs

```diff
@@ -58,13 +57,17 @@ public static class Program
 	/// </summary>
 	private static async Task Main(string[] args)
 	{
-		var argsProcessor = new ArgsProcessor(args);
+		ArgsProcessor argsProcessor = new(args);
 
-		// For now this is enough. If you run it on macOS you want to sign.
-		if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+		if (argsProcessor.IsMacSigning())
```

Comment:
Are you running this on mac? I think the only purpose of running the packager on mac can be that we are signing.

Reply:
Also, the sign is kind of confusing because we have another sign procedure at the end.

Reply:
Until no reason pls revert back.

Reply:
So the reason is simply that this allows to run `dotnet build && dotnet run --onlybinaries` on macOS, consequently Wasabi.Packager is much more multiplatform friendly. AFAIK Jumar (today) and Lucas (many months ago) expressed their wish to be able to use Packager on their machines.

> Also, the sign is kind of confusing because we have another sign procedure at the end.

We are talking about a simple check here and it looks like it helps Packager to be much more multiplatform (for no cost really). So can we come up with a better name to have the benefits? Or am I missing something?

Reply:
Out of the scope of the PR and breaking change for the release procedure. The packager will be never able to run on mac (because of WSL). However, making "onlybinaries" work sense is some very special cases for the devs. I added a script file for myself to make the release procedure quick.

Reply:
Currently "onlybinaries" is not working on mac. It fails with an error.

Reply:
The problem was the missing restore. When I added it back it is fixed.